README.md

```diff
@@ -1,2 +1,82 @@
-# wdio-browserstack-service
-Browserstack-Webdriverio service integration plugin: https://browserstack.atlassian.net/wiki/spaces/ENG/pages/3415278082/WebdriverIO+BrowserStack+Service?focusedCommentId=3418718374#comment-3418718374
+WebdriverIO Browserstack Service
+==========
+
+> A WebdriverIO service that manages local tunnel and job metadata for Browserstack users.
+
+## Installation
+
+
+The easiest way is to keep `@browserstack/wdio-browserstack-service` as a devDependency in your `package.json`.
+
+```json
+{
+    "devDependencies": {
+        "@browserstack/wdio-browserstack-service": "^1.0.0"
+    }
+}
+```
+
+You can simple do it by:
+
+```sh
+npm install @browserstack/wdio-browserstack-service --save-dev
+```
+
+Instructions on how to install `WebdriverIO` can be found [here.](https://webdriver.io/docs/gettingstarted)
+
+
+## Configuration
+
+WebdriverIO has Browserstack support out of the box. You should simply set `user` and `key` in your `wdio.conf.js` file. This service plugin provides supports for [Browserstack Tunnel](https://www.browserstack.com/automate/node#setting-local-tunnel). Set `browserstackLocal: true` also to activate this feature.
+Reporting of session status on BrowserStack will respect `strict` setting of Cucumber options.
+
+```js
+// wdio.conf.js
+export.config = {
+    // ...
+    user: process.env.BROWSERSTACK_USERNAME,
+    key: process.env.BROWSERSTACK_ACCESS_KEY,
+    services: [
+        ['@browserstack/wdio-browserstack-service', {
+            browserstackLocal: true
+        }]
+    ],
+    // ...
+};
+```
+
+## Options
+
+In order to authorize to the BrowserStack service your config needs to contain a [`user`](https://webdriver.io/docs/options#user) and [`key`](https://webdriver.io/docs/options#key) option.
+
+### browserstackLocal
+Set this to true to enable routing connections from Browserstack cloud through your computer. You will also need to set `browserstack.local` to true in browser capabilities.
+
+Type: `Boolean`<br />
+Default: `false`
+
+### preferScenarioName
+Cucumber only. Set this to true to enable updating the session name to the Scenario name if only a single Scenario was ran. Useful when running in parallel with [wdio-cucumber-parallel-execution](https://github.com/SimitTomar/wdio-cucumber-parallel-execution).
```

Comment:
The default should be True? What will be the ScenarioName if it will be false?

Reply:
gets called with name of feature is disabled